.circleci/config.yml

```diff
@@ -6,32 +6,36 @@ executors:
       goversion:
         type: string
         default: "11"
-    working_directory: /go/src/github.com/honeycombio/beeline-go
+    working_directory: /home/circleci/go/src/github.com/honeycombio/beeline-go
     docker:
-      - image: circleci/golang:1.<< parameters.goversion >>
+      - image: cimg/go:1.<< parameters.goversion >>
+        environment:
+          GO111MODULE: "on"
 
 jobs:
   test_beeline:
     parameters:
       goversion:
         type: string
-        default: "11"
+        default: "12"
     executor:
       name: go
       goversion: "<< parameters.goversion >>"
     steps:
       - checkout
-      - run: go get -v -t -d ./...
-      - run: go test -race -v ./...
+      - run:
+          name: go-get
```

Comment:
I think if you want to ensure in the build that PRs include any necessary updates to go.mod/go.sum, you'll want to run `go vet -mod=readonly ./...` before this?  (Or just skip the go-get stage and run `go test -mod=readonly -race -v ./...` as the single stage? I'll confess I'm not 100% clear on when `go get` does and does not happen transitively.)